x-pack/plugins/transform/common/constants.ts

```diff
@@ -43,6 +43,7 @@ export const TRANSFORM_REACT_QUERY_KEYS = {
   GET_TRANSFORM_NODES: 'transform.get_transform_nodes',
   GET_TRANSFORM_AUDIT_MESSAGES: 'transform.get_transform_audit_messages',
   GET_TRANSFORM_STATS: 'transform.get_transform_stats',
+  GET_TRANSFORMS_STATS: 'transform.get_transforms_stats',
```

Comment:
With the switch to do individual requests this key is no longer used within the client side code so could be removed.

Reply:
Reverted this update now that we are refeteching all stats together again